### DISCUSSION TITLE
Search Cache Needed

### DISCUSSION THREAD

**Discussion by sanga549** [keepassxreboot/keepassxc#9587] [Q&A] [answered]
I'm a convert from keepass to keepassxc. I'm used to having a Search Cache in Keepass where I could click the Search drop-down and select one of my previous search terms. This is handy when I'm searching through records grabbing data. I noticed in KeepassXC there isn't a Search Cache drop-down so I have to keep retyping all my search terms during my session which is cumbersome.  

Is the Search Cache there and I'm missing it or does it not exist. If not exist, it sure would be a helpful feature request to add. -Thanks!

**Comment by droidmonkey** [comment, marked as answer]
You're in luck, we have that feature (even better). Make sure you are updated to version 2.7.5. Then you will see a save icon in the search bar where you can save your frequently used searches. Recall them at any time in the searches and tags panel in the lower left.

**Reply by sanga549** [reply]
I saw the save icon but didn't see where it went or how to recall it. I see in the lower left corner now. Thank you so much! :)

**Comment by sanga549** [comment]
Hi, while saving frequent searches is a nice option and I tried it however often times my search can be random depending on what I'm looking for in a single session. 

Example. I look for "Book", get what I need then look for someone's name "Smith". Add a few more random search words I may use during my session. Then I may need recall my "Book" results so I have to type "Book" again.  In KeePass, recent searches were cached in a drop-down so I could just open the drop-down and pick "Book". Random search words such as this not worth saving because they're on-off searches. This cache clears after I logout of Keepass but it does come in handy during session.  I don't think KeePassXC has this capability, does it?

**Reply by droidmonkey** [reply]
No we don't. That would be a feature request.
